web/api/api.py

```diff
@@ -200,7 +200,7 @@ def update_plugin_category(slug, category):
     if not plugin:
         return api_util.api_not_found('No plugin with slug %s' % slug)
 
-    if not category in [c['id'] for c in get_all_categories_cached()]:
+    if category not in [c['id'] for c in get_all_categories_cached()]:
```

Comment:
This is not related to the change, but it's a great place to use a generator expression instead of a list comprehension for (potential) speed and memory improvements.